pkgs/build-support/kernel/make-initrd.sh

```diff
@@ -39,7 +39,7 @@ mkdir -p $out
 for PREP in $prepend; do
   cat $PREP >> $out/initrd
 done
-(cd root && find * -print0 | cpio -o -H newc -R 0:0 --null | perl $cpioClean | $compressor >> $out/initrd)
+(cd root && find * -print0 | sort -z | cpio -o -H newc -R 0.0 --null | perl $cpioClean | $compressor > $out/initrd)
```

Comment:
Why is `>>` changed to `>` here? Also, why change `:` to `.` (though this shouldn't change semantics)? BTW, from `cpio` documentation:

```
              To  avoid  the lookup and ensure that arguments are treated as numeric values, prefix them with a plus
              sign, e.g.: -R +0:+0.
```

Can't hurt either.